spec/break_spec.lua

```diff
@@ -1,6 +1,78 @@
+  nnode({ text ="her", height =  7.56836, depth = 0.14647, width = 14.09668 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="under", height =  7.56836, depth = 0.14647, width = 24.59473 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="any", height =  4.6875, depth = 2.44139, width = 15.03906 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="other", height =  7.56836, depth = 0.14647, width = 22.56836 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="name", height = 4.6875, depth = 0.1953, width = 25.04883 })
+  glue({ width = SILE.length.new({ length = 2.93619, stretch = 3.30322, shrink = 0.24467 }) })
+  nnode({ text ="In", height =  6.15234, depth = 0.0, width = 8.4961 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="his", height =  7.56836, depth = 0.14647, width = 12.08984 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="eyes", height =  4.6875, depth = 2.44139, width = 17.83691 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="she", height =  7.56836, depth = 0.14647, width = 13.99902 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="eclipses", height =  7.56836, depth = 2.34373, width = 31.9043 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="and", height =  7.56836, depth = 0.14647, width = 15.30762 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="predominates", height =  7.56836, depth = 2.34373, width = 56.7334 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="the", height =  7.56836, depth = 0.14647, width = 13.5791 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="whole", height =  7.56836, depth = 0.14647, width = 24.93652 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="of", height =  7.56836, depth = 0.14647, width = 8.13965 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="her", height =  7.56836, depth = 0.14647, width = 14.09668 })
+  glue({ width = SILE.length.new({ length = 2.20215, stretch = 1.10107, shrink = 0.73404 }) })
+  nnode({ text ="sex.", height =  4.6875, depth = 0.1953, width = 15.6543 })
+
+  it("should sleauth the right break point", function()
```

Comment:
Should what?

Reply:
Minus the extraneous 'a':

> [sleuth](https://www.dictionary.com/browse/sleuth)
>
> &nbsp;&nbsp;verb (used with or without object)
> &nbsp;&nbsp;&nbsp;&nbsp;to track or trail, as a detective.

Living long term in a non-English speaking country and speaking a second non-English language in the home has ruined my English spelling.

Reply:
I tacked this fix onto a marginally related commit in #754.